### ISSUE TITLE
ImportError: cannot import name 'makeMappingArray' from 'matplotlib.colors'

### ISSUE THREAD

**Issue by dmartintucker** [minimaxir/stylecloud#23]
Attempting to run your example code:
```
import stylecloud
stylecloud.gen_stylecloud(file_path='constitution.txt')
```

I get the following traceback error:
```
Traceback (most recent call last):
  File "/Users/d/Documents/word-clouds/generate_clouds.py", line 1, in <module>
    import stylecloud
  File "/usr/local/lib/python3.9/site-packages/stylecloud/__init__.py", line 1, in <module>
    from .stylecloud import gen_stylecloud
  File "/usr/local/lib/python3.9/site-packages/stylecloud/stylecloud.py", line 6, in <module>
    from matplotlib.colors import makeMappingArray, to_rgb
ImportError: cannot import name 'makeMappingArray' from 'matplotlib.colors' (/usr/local/lib/python3.9/site-packages/matplotlib/colors.py)
```

Any suggestions on how to remedy this would be greatly appreciated!

**Comment by Rachit-Singh** [comment]
This module named "makeMappingArray" is deprecated since version 3.2. So as a bodge, revert back to version 3.2.

PS: Before starting a new issue, please see if the same issue has been raised before. #22 is the same and has been answered by me already.

**Comment by dmartintucker** [comment]
Much appreciated! Apologies for the repeat issue.

**Comment by minimaxir** [comment]
I just pushed out v0.5.2 which removes `makeMappingArray` and replaces it with a more correct implementation. Let me know if that works! (closing myself since dupe)
